index.html

```diff
@@ -0,0 +1,124 @@
+          home. Search Engine Optimization (SEO) allows you to increase your
+          visibility and find the right customers for your business.
+        </p>
+      </section>
+      <section id="online-reputation-management">
+        <img
+          class="float-right-img"
+          src="./assets/images/online-reputation-management.jpg"
+          alt="A person seeing some reputation increase graphics on a laptop and using a cell phone"
+        />
+        <h2>Online Reputation Management</h2>
+        <p>
+          The web is full of opinions, and some of these can be negative. Social
+          media allows anyone with an internet connection to say whatever they
+          want about your business. Online Reputation Management gives you the
+          control over what potential customers see when they search for your
+          business.
+        </p>
+      </section>
+      <section id="social-media-marketing">
+        <img
+          class="float-left-img"
+          src="./assets/images/social-media-marketing.jpg"
+          alt="A group of people using their laptops and sharing ideas around a table"
+        />
+        <h2>Social Media Marketing</h2>
+        <p>
+          Social media continues to have a sizable influence on buying habits.
+          Social media marketing helps you determine which platforms are suited
+          to your brand, using analytics to find the right markets and increase
+          your lead generation.
+        </p>
+      </section>
+    </main>
+
+    <!--This is the Benefits section with some extra content of the marketing website -->
+    <aside class="benefits">
+      <section>
+        <h3>Lead Generation</h3>
+        <img
+          src="./assets/images/lead-generation.png"
+          alt="An engine passing through a funnel and becoming a dollar."
+        />
+        <p>
+          Inbound strategies for lead generation require less work for your
+          business, bringing customers directly to your website.
+        </p>
+      </section>
+      <section>
+        <h3>Brand Awareness</h3>
+        <img
+          src="./assets/images/brand-awareness.png"
+          alt="A person with suit and tie with a bulb lamp as his head with some awareness symbol."
+        />
+        <p>
+          Users find your business through paid and organic searches, increasing
+          the search ranking and visibility for your business.
+        </p>
+      </section>
+      <section>
+        <h3>Cost Management</h3>
+        <img
+          src="./assets/images/cost-management.png"
+          alt="An engine behind some coins with some dollars inside."
+        />
+        <p>
+          As the search ranking for your business increases, your advertising
+          costs decrease, and you no longer need to advertise your page.
+        </p>
+      </section>
+    </aside>
+
+    <footer>
+      <h2>Made with ❤️️ by Horiseon</h2>
```

Comment:
Being really picky here as your code is very good. This could of been a `h4` as this would follow the order of the `h` tags within the code, however, as the footer is it's own section, a `h2` works fine.

Reply:
Changed. Thanks for this comment. Makes sense as the code will be looking more semantic.